PROTOCOL.md

```diff
@@ -679,13 +679,14 @@ Field Name | Descrption
 shareCredentialsVersion | The file format version of the profile file. This version will be increased whenever non-forward-compatible changes are made to the profile format. When a client is running an unsupported profile file format version, it should show an error message instructing the user to upgrade to a newer version of their client.
 endpoint | The url of the sharing server.
 bearerToken | The [bearer token](https://tools.ietf.org/html/rfc6750) to access the server.
-
+expirationTime | The expiration time of the bearer token. This field is optional.
 Example:
 
 ```
 {
   "shareCredentialsVersion": 1,
   "endpoint": "https://sharing.delta.io/delta-sharing/",
-  "bearerToken": "<token>"
+  "bearerToken": "<token>",
+  "expirationTime": "9999-12-31 23:59:59.999"
```

Comment:
```suggestion
  "expirationTime": "2021-11-12T00:12:29Z"
```

Could you also update other hard code strings in tests using ISO 8601 format?

Reply:
done.